tables/uga.utb

```diff
@@ -29,13 +29,14 @@
 # liblouis  comes with ABSOLUTELY NO WARRANTY.
 
 
-#The braille code for Ugaritic is innovative, but closely follows the Cuneiform represented at ORACC. The braille code for Ugaritic is represented in braille as the name for the sign in Ugaritic. Thus, if the Ugaritic sign is a "Alepha," then the braille code for this sign would be a dot 1.
+#The braille code for Ugaritic is innovative, but accurately represents the Cuneiform . The braille code for Ugaritic is represented in braille as the name for the sign in Ugaritic. Thus, if the Ugaritic sign is a "Alepha," then the braille code for this sign would be a dot 1, similar to Hebrew.
```

Comment:
Mentioning of "ORACC" was dropped from this comment, as well as from the other tables that used to mention it. Is it not relevant? We do have "oracc" in the `#+system` metadata.